<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RocketController</class>
 <widget class="QWidget" name="RocketController">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1400</width>
    <height>1050</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="mainLayout" stretch="3,4,9,2">
     <property name="spacing">
      <number>50</number>
     </property>
     <item alignment="Qt::AlignmentFlag::AlignHCenter">
      <widget class="QLabel" name="label">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <family>Segoe UI Black</family>
         <pointsize>60</pointsize>
         <italic>true</italic>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">
	border-radius: 25px;
	border: 5px solid rgb(212,235,242);
	background-color: white;
	color: black;


</string>
       </property>
       <property name="text">
        <string>  Estaci√≥n de Tierra by LEEM  </string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QGridLayout" name="displaysLayout">
       <property name="sizeConstraint">
        <enum>QLayout::SizeConstraint::SetNoConstraint</enum>
       </property>
       <item row="1" column="0" alignment="Qt::AlignmentFlag::AlignHCenter">
        <widget class="QLabel" name="statusLCD">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="font">
          <font>
           <family>Segoe UI Semibold</family>
           <pointsize>20</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color: black;
</string>
         </property>
         <property name="text">
          <string>Standby</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLCDNumber" name="timeLCD">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="styleSheet">
          <string notr="true">QLCDNumber
{
	color: white;
}

QFrame
{
	border: 0px, white;
	border-radius: 25px;
}</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1" alignment="Qt::AlignmentFlag::AlignHCenter">
        <widget class="QLabel" name="timeLabel">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="font">
          <font>
           <family>Segoe UI Black</family>
           <pointsize>20</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color: black;
</string>
         </property>
         <property name="text">
          <string>Time [s]</string>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QLCDNumber" name="altitudLCD">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="styleSheet">
          <string notr="true">QLCDNumber
{
	color: white;
}

QFrame
{
	border: 0px, white;
	border-radius: 25px;
}</string>
         </property>
        </widget>
       </item>
       <item row="0" column="3" alignment="Qt::AlignmentFlag::AlignHCenter">
        <widget class="QLabel" name="maxAltitudLabel">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="font">
          <font>
           <family>Segoe UI Black</family>
           <pointsize>20</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color: black;
</string>
         </property>
         <property name="text">
          <string>Apogee [m]</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2" alignment="Qt::AlignmentFlag::AlignHCenter">
        <widget class="QLabel" name="altitudLabel">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="font">
          <font>
           <family>Segoe UI Black</family>
           <pointsize>20</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color: black;
</string>
         </property>
         <property name="text">
          <string>Altitude [m]</string>
         </property>
        </widget>
       </item>
       <item row="1" column="3">
        <widget class="QLCDNumber" name="maxAltitudLCD">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="styleSheet">
          <string notr="true">QLCDNumber
{
	color: white;
}

QFrame
{
	border: 0px, white;
	border-radius: 25px;
}</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0" alignment="Qt::AlignmentFlag::AlignHCenter">
        <widget class="QLabel" name="statusLabel">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="font">
          <font>
           <family>Segoe UI Black</family>
           <pointsize>20</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color: black;
</string>
         </property>
         <property name="text">
          <string>Phase</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QGridLayout" name="gaugesLayout">
       <property name="sizeConstraint">
        <enum>QLayout::SizeConstraint::SetNoConstraint</enum>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="horizontalSpacing">
        <number>0</number>
       </property>
       <property name="verticalSpacing">
        <number>40</number>
       </property>
       <item row="1" column="1">
        <widget class="QQuickWidget" name="gyroscope">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="resizeMode">
          <enum>QQuickWidget::ResizeMode::SizeRootObjectToView</enum>
         </property>
        </widget>
       </item>
       <item row="0" column="1" alignment="Qt::AlignmentFlag::AlignHCenter">
        <widget class="QLabel" name="gyroscopeLabel">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="font">
          <font>
           <family>Segoe UI Black</family>
           <pointsize>20</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color: black;
</string>
         </property>
         <property name="text">
          <string>Gyroscope</string>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QQuickWidget" name="acc">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="resizeMode">
          <enum>QQuickWidget::ResizeMode::SizeRootObjectToView</enum>
         </property>
        </widget>
       </item>
       <item row="0" column="0" alignment="Qt::AlignmentFlag::AlignHCenter">
        <widget class="QLabel" name="speedLabel">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="font">
          <font>
           <family>Segoe UI Black</family>
           <pointsize>20</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color: black;
</string>
         </property>
         <property name="text">
          <string>Speed [m/s]</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QQuickWidget" name="speed">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="resizeMode">
          <enum>QQuickWidget::ResizeMode::SizeRootObjectToView</enum>
         </property>
        </widget>
       </item>
       <item row="0" column="2" alignment="Qt::AlignmentFlag::AlignHCenter">
        <widget class="QLabel" name="accLabel">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="font">
          <font>
           <family>Segoe UI Black</family>
           <pointsize>20</pointsize>
           <italic>false</italic>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color: black;
</string>
         </property>
         <property name="text">
          <string>Acceleration [g]</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QGridLayout" name="buttonLayout">
       <property name="horizontalSpacing">
        <number>40</number>
       </property>
       <property name="verticalSpacing">
        <number>30</number>
       </property>
       <item row="0" column="1">
        <widget class="QPushButton" name="phase_button">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="font">
          <font>
           <family>Consolas</family>
           <pointsize>16</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton 
{
	border-radius: 15px;
	height: 30px;
	background-color: white;
}
</string>
         </property>
         <property name="text">
          <string>Phase</string>
         </property>
         <property name="checkable">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="0" column="3">
        <widget class="QPushButton" name="alarm_button">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="font">
          <font>
           <family>Consolas</family>
           <pointsize>16</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton 
{
	border-radius: 15px;
	height: 30px;
	background-color: white;
}
</string>
         </property>
         <property name="text">
          <string>Alarm on</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QPushButton" name="tare_button">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="font">
          <font>
           <family>Consolas</family>
           <pointsize>16</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton 
{
	border-radius: 15px;
	height: 30px;
	background-color: white;
}
</string>
         </property>
         <property name="text">
          <string>Tare</string>
         </property>
         <property name="checkable">
          <bool>false</bool>
         </property>
         <property name="autoDefault">
          <bool>false</bool>
         </property>
         <property name="default">
          <bool>false</bool>
         </property>
         <property name="flat">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QPushButton" name="ping_button">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="font">
          <font>
           <family>Consolas</family>
           <pointsize>16</pointsize>
          </font>
         </property>
         <property name="autoFillBackground">
          <bool>false</bool>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton 
{
	border-radius: 15px;
	height: 30px;
	background-color: white;
}
</string>
         </property>
         <property name="text">
          <string>Ping</string>
         </property>
         <property name="checkable">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QQuickWidget</class>
   <extends>QWidget</extends>
   <header location="global">QtQuickWidgets/QQuickWidget</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
 <slots>
  <slot>teston()</slot>
  <slot>ledon()</slot>
  <slot>ledoff()</slot>
  <slot>ignitionon()</slot>
  <slot>ignitionoff()</slot>
  <slot>paracason()</slot>
  <slot>paracasoff()</slot>
  <slot>alarmaon()</slot>
  <slot>alarmaoff()</slot>
  <slot>testoff()</slot>
  <slot>exporttest()</slot>
  <slot>exportgraph()</slot>
 </slots>
</ui>
